src/Framework/Framework/Controls/HtmlGenericControl.cs

```diff
@@ -413,7 +413,7 @@ private void AddHtmlAttributesToRender(ref RenderState r, IHtmlWriter writer)
 
                 if (knockoutExpression is {})
                 {
-                    if (attributeName.Equals("class", StringComparison.OrdinalIgnoreCase))
+                    if (attributeName == "class")
```

Comment:
Maybe this line does not need to be reverted., otherwise it's fine

Reply:
You are right. I did not realize this - just performed a `git revert` and did not realize that this should definitely stay